schema.sql

```diff
@@ -2,4 +2,9 @@
 
 CREATE TABLE animals (
     name varchar(100)
+    id INT,
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
 );
```

Comment:
- [OPTIONAL]  I suggest to make the id column not null and autoincremented 👍
```sql
id int NOT NULL IDENTITY(1,1)
```

[OPTIONAL] I suggest to mark the id column as primary key 💯
```sql
PRIMARY KEY (id)
```